File: src/console/log/output/DefaultOutputHandler.php
<?php

namespace pocketcloud\cloud\console\log\output;

use pocketcloud\cloud\console\Console;
use pocketcloud\cloud\console\log\logger\ILogger;

final class DefaultOutputHandler implements OutputHandler {

    public function shouldOutput(ILogger $logger): bool {
        return true;
    }

    public function handleOutput(string $message): void {
        Console::getInstance()->println($message);
    }
}

File: cloud-core/src/main/java/de/pocketcloud/cloud/console/output/impl/DefaultOutputHandler.java
package de.pocketcloud.cloud.console.output.impl;

import de.pocketcloud.api.logging.ILogger;
import de.pocketcloud.cloud.PocketCloud;
import de.pocketcloud.cloud.console.output.OutputHandler;
import org.jetbrains.annotations.NotNull;

public final class DefaultOutputHandler implements OutputHandler {

    @Override
    public boolean canOutput(@NotNull ILogger logger) {
        return true;
    }

    @Override
    public void handleOutput(String message) {
        PocketCloud.instance().console().print(message);
    }
}